modules/installation-gcp-marketplace.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-marketplace-_{context}"]
+= Using a GCP Marketplace image
+If you are deploying an {product-title} cluster using a GCP Marketplace image, create the manifests and edit the compute machine set definitions to specify the GCP Marketplace image.
+
+.Prerequisites
+
+* You have the {product-title} installation program and the pull secret for your cluster.
+
+.Procedure
+
+. Generate the installation manifests by running the following command.
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_dir>
+----
+
+. Locate the following files:
+
+** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-0.yaml`
+** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-1.yaml`
+** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-2.yaml`
+
+. In each file, edit the `.spec.template.spec.providerSpec.value.disks[0].image` property to `projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145`
+
+.Example compute machine set with the GCP Marketplace image
```

Comment:
This code block should be indented to be with step 3, correct?

Reply:
No. Not necessarily. If there was a step 4, I would have indented it. But I believe it is OK as is with the label.